app/src/main/java/de/danoeh/antennapod/fragment/ItemFragment.java

```diff
@@ -217,7 +217,7 @@ private void showOnDemandConfigBalloon(boolean offerStreaming) {
             balloon.dismiss();
         });
         negativeButton.setOnClickListener(v1 -> {
-            UsageStatistics.askAgainLater(UsageStatistics.ACTION_STREAM); // Type does not matter. Both are silenced.
+            UsageStatistics.doNotAskAgain(UsageStatistics.ACTION_STREAM); // Type does not matter. Both are silenced.
```

Comment:
The `askAgainLater` method is now unused, I think, so it can be deleted.

Reply:
I left it since UsageStatistics looks like it was built to support other statistics in the future. If `askAgainLater()` is not required, the class could be simplified, using only one `_hidden` preference set by the _No_ button instead of now two.

Reply:
That class is a bit over-engineered :D I guess when we want to add more on-demand configuration options later, we should just do the same and stop asking when users select "no". Would be great if you could simplify the class :)